lib/active_merchant/billing/gateways/quickbooks.rb

```diff
@@ -267,6 +269,10 @@ def extract_response_body_or_raise(response_error)
         end
         response_error.response.body
       end
+
+      def refund_or_void_uri(authorization)
```

Comment:
voids and refunds hit the same endpoint - once we get confirmation from Intuit that they work in sandbox, it will look like this. YAGNI?